pkg/defaults/defaults_test.go

```diff
@@ -42,6 +42,9 @@ func TestStepConfigsForBuild(t *testing.T) {
 	noopResolver := func(root, cache *api.ImageStreamTagReference) (*api.ImageStreamTagReference, error) {
 		return root, nil
 	}
+	rootSources := []api.ImageStreamSource{api.NewImageStreamSource(api.ImageStreamSourceRoot, nil)}
```

Comment:
nit: in my personal mental framework for tests, there's no such thing as "DRY" and making separate, independent tests depend on the same construct is an antipattern as it reduces failure specificity 

(however, if we do not have a pointer to a slice in the sources member you can just inline these, which would fix it)